portal/src/components/Collections/Collections.component.js

```diff
@@ -17,31 +17,30 @@ export default {
     },
     editableContent: {
       type: Boolean,
-      default: false
+      default: true
```

Comment:
dit maakt niet andere componenten per ongeluk stuk?

Reply:
ik denk dat dit per ongeluk is gegaan. good catch